examples/http-python3.10/README.md

```diff
@@ -48,13 +45,24 @@ To close the Unikraft instance, use:
 kraft rm nostalgic_snowflake
 ```
 
-Note that closing the `kraft run` command (such as using `Ctrl+c`) does not close the Unikraft instance.
+Note that closing the `kraft run` command (e.g., via `Ctrl+c`) does not kill the Unikraft instance.
 If you want the Unikraft instance closed when closing the `kraft run` command, use the `--rm` option:
 
 ```bash
 kraft run --rm -p 8080:8080 --plat qemu --arch x86_64
 ```
 
+Finally, note that depending on how you modify this example your instance *may* need more memory to run.
```

Comment:
Use `**may**` for bold instead of `*may*` for italic.

Reply:
will fix.